src/main/java/ru/yandex/practicum/filmorate/model/Film.java

```diff
@@ -0,0 +1,23 @@
+package ru.yandex.practicum.filmorate.model;
+
+import lombok.Data;
+import ru.yandex.practicum.filmorate.validation.constraints.FilmReleaseDateConstraint;
+
+import javax.validation.constraints.*;
+import java.time.LocalDate;
+
+@Data
+public class Film {
+    private Integer id;
+    @NotBlank
+    private String name;
+    @Size(max = 200, message = "Description length must be not more 200 characters")
```

Comment:
@Size(min = 1, max = 200, message = "Description length must be not more 200 characters")